pkgs/by-name/re/regrippy/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "regippy";
+  version = "2.0.1";
+  doCheck = false;
+
+  src = fetchFromGitHub {
+    owner = "airbus-cert";
+    repo = "regrippy";
+    rev = "05c9db999853c47af1d15f92f1a34aa2441e8882";
+    hash = "sha256-gS7qVPlXwn6UXRXPN5ahPmQL3JpwmESUEi0KKAzOo+8=";
+  };
+
+  propagatedBuildInputs = [
+    python3.pkgs.importlib-resources
+    python3.pkgs.pytest
+    python3.pkgs.python-registry
+    python3.pkgs.setuptools
+  ];
+
+  meta = {
+    description = "A modern Python-3-based alternative to RegRipper";
+    homepage = "https://github.com/airbus-cert/regrippy";
+    license = lib.licenses.asl20;
+    platforms = lib.platforms.all;
+    maintainers = with lib.maintainers; [ mikehorn ];
+    mainProgram = "regrip";
```

Comment:
There is no executable called `regrip`, maybe consider:
- set mainProgram to `regrip.py`
- create the `regrip` executable, called `regrip.py` under the hood